### DISCUSSION TITLE
Always on top

### DISCUSSION THREAD

**Discussion by quakec** [WerWolv/ImHex#1493] [Feedback]
Firstly, can I just start with; how awesome is this hex editor, it's everything I've been looking for and more! I love it.  If I hadn't found this, I'd have probably done what you've done here and wrote my own.

Okay, so I can't find an option (or it does't exist) to set the window state to always on top (you know the one that uses SetWindowPos(...) api call from user32.dll).  Personally I would expect this to be under the view menu, much like Notepad++ :)

Would greatly appreciate this feature, as many others would I imagine.

![image](https://github.com/WerWolv/ImHex/assets/1788271/2a08dd01-80ef-4698-b0eb-62a11de75800)

**Comment by WerWolv** [comment]
Hey, thank you! Latest commit implements both always on top and fullscreen mode :) 82a3017629c25f5ce59dda062fccc1462dca3779

![image](https://github.com/WerWolv/ImHex/assets/10835354/844ae81e-b12c-4ce8-8103-9e8d9e00b7e9)
